int8 numSensors

---

bool setSensors
